dcos/package.py

```diff
@@ -1437,3 +1437,18 @@ def as_dict(self):
         """
 
         return {'source': self.source.url, 'packages': self.packages}
+
+
+def get_apps_for_framework(framework_name, client):
+    """ Return all apps running the given framework.
+
+    :param framework_name: framework name
+    :type framework_name: str
+    :param client: marathon client
+    :type client: marathon.Client
+    :rtype: [dict]
+    """
+
+    return [app for app in client.get_apps()
+            if app.get('labels', {}).get(
+                PACKAGE_FRAMEWORK_NAME_KEY) == framework_name]
```

Comment:
Looks like that was added back in 2014: https://github.com/mesosphere/marathon/commit/8fd01a83b1958e6ea95c842bc84b580bd1285e35#diff-268ca9c8f5855c658cfb04e862b14d2bR372

It's tagged with `v0.7.0`, so we should be fine using it since we tell users marathon is outdated if its not `0.8.1` or above